ui/pages/confirmations/components/confirm/splash/smart-account-update/smart-account-update.tsx

```diff
@@ -0,0 +1,181 @@
+          <Text
+            color={TextColor.textAlternative}
+            variant={TextVariant.bodyMd}
+            marginInlineEnd={2}
+          >
+            {t('smartAccountRequestFor')}
+          </Text>
+          <Name
+            value={from}
+            type={NameType.ETHEREUM_ADDRESS}
+            preferContractSymbol
+            variation={chainId}
+          />
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/speedometer.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountBetterTransaction')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountBetterTransactionDescription')}
+            </Text>
+          </Box>
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/petrol-pump.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountPayToken')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountPayTokenDescription')}
+            </Text>
+          </Box>
+        </Box>
+        <Box display={Display.Flex} alignItems={AlignItems.flexStart}>
+          <img width="24px" src="./images/sparkle.svg" />
+          <Box
+            display={Display.Flex}
+            flexDirection={FlexDirection.Column}
+            marginInlineStart={2}
+          >
+            <Text
+              color={TextColor.textDefault}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Medium}
+            >
+              {t('smartAccountFeatures')}
+            </Text>
+            <Text
+              color={TextColor.textAlternative}
+              variant={TextVariant.bodyMd}
+              fontWeight={FontWeight.Normal}
+            >
+              {t('smartAccountFeaturesDescription')}
+              <a
+                key="learn_more_link"
+                href={SMART_ACCOUNT_INFO_LINK}
+                rel="noreferrer"
+                target="_blank"
+              >
+                {t('learnMore')}
```

Comment:
Looks like this needs to be updated

"Keep the same account address, and you can switch back anytime.[learn more](https://support.metamask.io/configure/accounts/what-is-a-smart-account),."

- missing space between period and learn
- learn should be capitalized
- there is a comma that appears after learn more

![CleanShot 2025-04-09 at 11 23 03](https://github.com/user-attachments/assets/4047dd53-2276-4812-ae61-4318edca4d08)

Reply:
Good catch, I addressed this.